File: All Codes/URI_1010.java
import java.util.Scanner;

/**
 * IMPORTANT: O nome da classe deve ser "Main" para que a sua solução execute
 * Class name must be "Main" for your solution to execute El nombre de la clase
 * debe ser "Main" para que su solución ejecutar
 */
public class URI_1010 {
 
    public static void main(String[] args) {

         try (Scanner input = new Scanner(System.in)) {
            input.nextInt();
            int b = input.nextInt();
            float g = input.nextFloat();
            input.nextInt();
            int d = input.nextInt();
            float h = input.nextFloat();
            float e = (b * g) + (d * h);
            System.out.printf("VALOR A PAGAR: R$ %.2f\n",e);
        }
 
    }
}

File: All Codes/URI_1010.py
product_1 = [float(x) for x in input().split()]
product_2 = [float(x) for x in input().split()]

total = (product_1[1] * product_1[2]) + (product_2[1] * product_2[2])
print(f"VALOR A PAGAR: R$ {total:.2f}") 